src/Resources/skeleton/stimulus/Controller.tpl.php

```diff
@@ -0,0 +1,7 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
```

Comment:
what bout adding `/* stimulusFetch: 'lazy' */` ?
Ref a PR of mine on the demo repo https://github.com/symfony/demo/pull/1284/files#r770934137

Reply:
Agreed, we should just include `/* stimulusFetch: 'lazy' */` by default in the template. It's easy enough for the user to remove if they do not want it.

cc/ @weaverryan